articles/modules/ROOT/pages/how-neo4j-browser-bolt-routing.adoc

```diff
@@ -1,16 +1,16 @@
-= How Neo4j Browser uses bolt+routing
+= How Neo4j Browser uses the neo4j scheme
 :slug: how-neo4j-browser-bolt-routing
 :author: José Rocha
 :neo4j-versions: 3.5, 4.0, 4.1, 4.2, 4.3, 4.4
 
-There are two kinds of Bolt drivers specified by URI scheme: the *bolt://* scheme is used to create a direct driver and the *bolt+routing://* scheme is used to create a routing driver.
+There are two kinds of Bolt drivers specified by URI scheme: the *bolt://* scheme is used to create a direct driver and the *neo4j://* scheme is used to create a routing driver.
```

Comment:
I'm actually not sure when this was changed. Notice how the top of the entry specifies
 > :neo4j-versions: 3.5, 4.0, 4.1, 4.2, 4.3, 4.4
 
 So I think this should rather be driver version specific vs. server version specific.

Reply:
Right, this would be about the pre-4.x drivers.